tests/integration/test_services_auto_tagging_copilot_feedback.py

```
@@ -0,0 +1,1217 @@
+
+    def test_update_patterns(self, tmp_path: Path) -> None:
+        from file_organizer.models.suggestion_types import SuggestionType
+        from file_organizer.services.suggestion_feedback import FeedbackEntry, SuggestionFeedback
+
+        fb = SuggestionFeedback(feedback_file=tmp_path / "feedback.json")
+        entries = [
+            FeedbackEntry(
+                suggestion_id="u1",
+                suggestion_type=SuggestionType.RENAME,
+                action="accepted",
+                file_path="/tmp/f.txt",
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/f.txt"

(S108)